website/pages/blog/graphql-over-internet.mdx

```diff
@@ -0,0 +1,250 @@
+
+Like all of the libraries mentioned in the presentation - `graphql-sse` too is server agnostic - you
+can run it in any JavaScript environment!
+
+## WebSockets vs. SSEs
+
+There's an abundance of articles, tables and benchmarks on the internet comparing WebSockets to
+SSEs, so I'll just briefly touch the obvious.
+
+|               | WebSockets                                                                                                                                  | SSEs                                                                                                                                                                                                        |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Communication | Real-time and bi-directional                                                                                                                | Transported over simple HTTP instead                                                                                                                                                                        |
+| Firewalls     | Has troubles with corporate and outdated firewalls                                                                                          | No trouble with _any_ firewall doing packet inspection                                                                                                                                                      |
+| Latency       | Lower because of persisted TCP connection                                                                                                   | Can be higher sometimes because of multiplexing                                                                                                                                                             |
+| Retries       | Needs to be developed in user-land                                                                                                          | Browsers have built-in retry mechanisms                                                                                                                                                                     |
+| Support       | Browser native through [`WebSocket`](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket). No custom implementations for the browser | Browser native through [`EventSource`](https://developer.mozilla.org/en-US/docs/Web/API/EventSource). But, can be custom implemented on [`fetch()`](https://developer.mozilla.org/en-US/docs/Web/API/fetch) |
+
+## Presentation
+
+Check out the video of the related presentation that happened at the
```

Comment:
```suggestion
Check out my presentation from
```

I think it's more clickable if it's obvious that the video is from the article's author